testcafe/tests/pages/drive/drive-model-private.js

```diff
@@ -232,17 +265,21 @@ export default class PrivateDrivePage extends DrivePage {
 
   //@param { Array } filesIndexArray : Array of files index
   async deleteElementsByIndex(filesIndexArray) {
-    const rowCountStart = await this.getContentRowCount('Before')
+    let rowCountStart
+    if (!t.fixtureCtx.isVR) {
+      rowCountStart = await this.getContentRowCount('Before')
+    }
     await this.selectElements(filesIndexArray)
 
     await t
       .click(this.cozySelectionbarBtnDelete)
-      .expect(this.modalDelete.visible)
+      .expect(this.modalFooter.visible)
       .ok('Delete button does not show up')
-      .click(this.modalDeleteBtnDelete)
-    await t.takeScreenshot()
-    const rowCountEnd = await this.getContentRowCount('After')
-    await t.expect(rowCountEnd).eql(rowCountStart - filesIndexArray.length)
+      .click(this.modalSecondButton)
+    if (!t.fixtureCtx.isVR) {
+      const rowCountEnd = await this.getContentRowCount('After')
+      await t.expect(rowCountEnd).eql(rowCountStart - filesIndexArray.length)
```

Comment:
We're using `rowCountStart` here but it can be `undefined`, is that normal?

Reply:
`rowCountStart`and `rowCountEnd` are only defined and used when we don't use the screenshots as it was the way to check the expected files where removed, but with the screenshots we don't need those checks as the screenshots will show the files were removed

Reply:
Oh sorry my bad, I missed the second `if (!t.fixtureCtx.isVR) ` 👍